src/index.js

```diff
@@ -48,26 +49,59 @@ function reset(element) {
   element.innerHTML = '';
 }
 
-function fetchData() {
-  Promise.all([allCustomerDataAPI, bookingDataAPI, roomDataAPI])
-    .then((values) => {
-      createInstances(values);
-      renderUserDashboard();
-    })
-}
+function findUserName() {
+  let possibleUsernames = [];
+  for (let i = 1; i < 51; i++) {
+    possibleUsernames.push(`customer${i}`);
```

Comment:
Just think its an interesting way you've done this by strictly defining the possible usernames and checking them against the password, I definently went the more short-cutted way of just checking the last 2 characters and the password